client/src/components/Videos.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import staticData from "../exampleresponse.json";
+import AddVideoForm from "./AddVideoForm";
+import Search from "./Search";
+import SingleVideo from "./SingleVideo";
+import "./Videos.css";
+
+const Videos = () => {
+  const sortedVideosData = staticData.sort((a, b) => b.rating - a.rating);
```

Comment:
When this function `Videos` is rerendered, it will run the entire function top to bottom each time.

This means that line 9, the sorting, is _also_ going to be run each time. It seems to me you're only interested in sorting this once though, then storing it in the `videosData` state. 

Two choices for how to achieve this. 

1. Easiest way - just move line 9 up so that it comes before the `Videos` function
2. Neater way - you can initialise useState using a function like this:
```const [variable, setVariable] = useState(() => staticData.sort(...))```

This is called Lazy initial state and is explained here: [https://reactjs.org/docs/hooks-reference.html#usestate](https://reactjs.org/docs/hooks-reference.html#usestate)